Python/pystate.c

```diff
@@ -209,7 +209,8 @@ PyInterpreterState_New(void)
     memset(interp, 0, sizeof(*interp));
     interp->id_refcount = -1;
 
-    interp->config.struct_size = sizeof(PyConfig);
+    // TODO: Should this pass the API/ABI version from the parent interpreter?
+    interp->config.header_version = PY_VERSION_HEX;
```

Comment:
Depends whether this field will become a "behave like this version" (then yes) or just "interpreter this struct like this version (then no).

Reply:
Yeah, I figure it would only matter if it ever became the former, and in that case, this is one of the places that would need to change.